src/model.py

```diff
@@ -280,10 +289,11 @@ async def generate(self, request):
             response = pb_utils.InferenceResponse(
                 output_tensors=[triton_output_tensor], error=error
             )
-            response_sender.send(response)
+            response_sender.send(
+                response, flags=pb_utils.TRITONSERVER_RESPONSE_COMPLETE_FINAL
+            )
             raise e
         finally:
-            response_sender.send(flags=pb_utils.TRITONSERVER_RESPONSE_COMPLETE_FINAL)
             self.ongoing_request_count -= 1
```

Comment:
Just curious: What's the purpose of this line?

Reply:
Making sure that all the requests have been finished before unloading the model.